Running npm install causes issues with versions

# Overview

Trying to install packages with `npm install` gives error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   dev react@"^17.0.2" from the root project
npm ERR!   peerOptional react@"^16.8.0 || ^17.0.0" from @storybook/addon-actions@6.2.9
npm ERR!   node_modules/@storybook/addon-actions
npm ERR!     dev @storybook/addon-actions@"^6.2.9" from the root project
npm ERR!     @storybook/addon-actions@"6.2.9" from @storybook/addon-essentials@6.2.9
npm ERR!     node_modules/@storybook/addon-essentials
npm ERR!       dev @storybook/addon-essentials@"^6.2.9" from the root project
npm ERR!   26 more (@storybook/addon-essentials, @storybook/addon-links, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"15.x || 16.x || 16.4.0-alpha.0911da3" from @reach/router@1.3.4
npm ERR! node_modules/@storybook/api/node_modules/@reach/router
npm ERR!   @reach/router@"^1.3.4" from @storybook/api@6.2.9
npm ERR!   node_modules/@storybook/api
npm ERR!     @storybook/api@"6.2.9" from @storybook/addon-actions@6.2.9
npm ERR!     node_modules/@storybook/addon-actions
npm ERR!       dev @storybook/addon-actions@"^6.2.9" from the root project
npm ERR!       1 more (@storybook/addon-essentials)
npm ERR!     8 more (@storybook/addon-essentials, @storybook/addons, ...)
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```
Trying to install react before `npm install` gives same error.

I could install frictionless-components using the `--legacy-peer-deps` flag.

OS: Ubuntu 20.04
Node: 14.17.0
npm: 7.11.1

---

Please preserve this line to notify @roll (lead of this repository)

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,2 @@
+# https://github.com/storybookjs/storybook/issues/12983
+legacy-peer-deps = true

